config/samples/otel.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: otel-spinapp
+spec:
+  image: ghcr.io/spinkube/spin-operator/cpu-load-gen:20240311-163328-g1121986
+  executor: containerd-shim-spin
+  replicas: 1
+---
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinAppExecutor
+metadata:
+  name: containerd-shim-spin
+spec:
+  createDeployment: true
+  deploymentConfig:
+    runtimeClassName: wasmtime-spin-v2
+    installDefaultCACerts: true
+    otel:
+      endpoint: http://otel-collector-opentelemetry-collector.default.svc.cluster.local:4318
+
+      
```

Comment:
Nit: Some extra newline stuff going on here. Not super important (hence the nit), but I'm overly sensitive to this kind of stuff 🤣. Ultimately whitespace doesn't matter, but I find clean whitespace makes code easier to read and I also find that clean whitespace shows craftsmanship and care for your code. 🤷 just my 2¢.